### PR TITLE
Interface updates

### PR DESCRIPTION
**Pull request by bajo**

### CODE REVIEW

File: squirrel_attention/src/fixate_on_3D_point.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#include <math.h>
+
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+
+#include "std_msgs/Float64.h"
+#include <dynamixel_controllers/SetRelativePosition.h>
```
Comment (NadiaHG) [inline squirrel_attention/src/fixate_on_3D_point.cpp:7]: dynamixel_controllers? do we still need this?

File: squirrel_attention/src/AttentionFusion.cpp [hunk 2]
```diff
@@ -143,7 +143,21 @@ void AttentionFusion::legsCallback(const people_msgs::People& msg)
     observeMutex_.lock();
     reason_ = "leg";
     observeMutex_.unlock();
-  }
+
+    robotino_msgs::LookAtPosition srv;
+    srv.request.target = next_.point;
+    srv.request.why = reason_;
```
Comment (NadiaHG) [inline squirrel_attention/src/AttentionFusion.cpp:149]: for the current version `robotino_msgs` ( https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/robotino_msgs/srv/LookAtPosition.srv)  version:

`srv.request.reason = reason_;`

Comment (bajo) [reply squirrel_attention/src/AttentionFusion.cpp:149]: Seems some old code slipped through. Will send an update later.

File: squirrel_attention/src/fixate_on_3D_point.cpp (hunk 1 above)
Comment (bajo) [reply squirrel_attention/src/fixate_on_3D_point.cpp:7]: No, I will remove it in the next update.

**Comment by ipa-nhg** [comment]
@bajo news here?

**Comment by bajo** [comment]
@ipa-nhg I'll update this pr today. Sorry for the delay.

**Comment by ipa-nhg** [comment]
@bajo @mzillich is this still needed or the PR should be closed?

**Comment by ipa-nhg** [comment]
out-of-date

**Pull request closed without merging**